app/i18n/en/gen.php

```diff
@@ -160,6 +160,13 @@
 		'nothing_to_load' => 'There are no more articles',
 		'previous' => 'Previous',
 	),
+	'period' => array(
+		'day' => 'days',
```

Comment:
The keys don't need to match the content, but this singular/plural thing stands out a bit. :-P

Reply:
True! I will fix that.